core/src/main/java/com/spotify/metrics/core/RemoteMeter.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2016 Spotify AB.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.spotify.metrics.core;
+
+import com.codahale.metrics.Metric;
+
+/**
+ * Like a coda hale Meter, but remoter.
+ */
+public interface RemoteMeter extends Metric {
```

Comment:
I'm a bit concerned about `RemoteMeter` extending `Metric`. This would allow it to be registered in `SemanticMetricRegistry` which doesn't make much sense.